### PR TITLE
Enable CircleCI

### PR DESCRIPTION
**Pull request by brian-brazil**
Just testing this out atm.

### CODE REVIEW

**Comment by brian-brazil** [comment]
This is all working. I have confirmed this breaks on Java 8 code.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![CircleCI](https://circleci.com/gh/rollerderby/scoreboard.svg?style=svg)](https://circleci.com/gh/rollerderby/scoreboard)
```
Comment (frank-weinberg) [inline README.md]: Why is this included here? This file is directed at end users and most end users presumably have no idea what this is about.

Comment (brian-brazil) [reply README.md]: I'm used to the various build indicators being included at the top of a project's readme, as an indication that all is healthy. https://github.com/prometheus/prometheus#prometheus- for example

Comment (brian-brazil) [reply README.md]: What would you like to do here?

This is the fashion and a common way to indicate that a project cares about code quality, but I'm not wedded to it.

Comment (frank-weinberg) [reply README.md]: Given that this file is primarily directed at new users of CRG, many of whom come in with a fear of not being able to install and run it on their own, and that one goal of the file is to help them overcome this fear, I am worried that if the first thing they see in the file is something they don't understand, this will work against this goal. 

But I am not super invested here either. Maybe we should bring this to the dev group to get a few more opinions.

Comment (brian-brazil) [reply README.md]: The current readme duplicates content in the wiki, so overall I'd be inclined to have a big thing at the start pointing users to the wiki and then dev stuff from there in the readme.

Comment (frank-weinberg) [reply README.md]: The readme intentionally duplicates that content as not everybody has internet access in their venues.

Comment (brian-brazil) [reply README.md]: I'm just going to remove this for now.

Depending on place to keep two sets of docs in place is not likely to be very robust, using a single source would be better.
